config/routes.rb

```diff
@@ -34,4 +34,6 @@
   patch '/favorites/:pet_id', to: 'favorites#update'
   delete '/favorites', to: 'favorites#destroy'
 
+  get '/pets/:id/applications', to: 'pet_show_applications#index'
```

Comment:
this really should have been pet_applications controller, but we already used that name. pet_applications really should have been named user_applications.  just a minor annoyance that falls in the "refactor later if we have time" category...

Reply:
looking like we might not have time, but commit the lesson to memory.... like not naming anything "application". :)

Reply:
by the way, this line is the merge "conflict." there were no deletions, _just_ one line added. there _is_ no conflict.